docker-bake.hcl

```diff
@@ -47,7 +47,7 @@ target "default" {
       // renovate: datasource=docker versioning=loose
       "debian:bookworm-slim@sha256:b1a741487078b369e78119849663d7f1a5341ef2768798f7b7406c4240f86aef",
       // renovate: datasource=docker versioning=loose
-      "debian:bullseye-slim@sha256:849d9d34d5fe0bf88b5fb3d09eb9684909ac4210488b52f4f7bbe683eedcb851"
+      "debian:trixie-slim@sha256:c85a2732e97694ea77237c61304b3bb410e0e961dd6ee945997a06c788c545bb"
```

Comment:
Sadly I think we can't stop producing bullseye images yet unless they're fully discontinued